py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -2572,3 +2572,24 @@ def aten_ops_copy(
         src.dtype,
         force_layer=True,
     )
+
+
+@dynamo_tensorrt_converter(torch.ops.aten.any.default)
+@dynamo_tensorrt_converter(torch.ops.aten.any.dim)
+@dynamo_tensorrt_converter(torch.ops.aten.any.dims)
+def aten_ops_any(
+    ctx: ConversionContext,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return impl.reduce.any(
+        ctx,
+        target,
+        SourceIR.ATEN,
+        name,
+        args[0],
+        args_bounds_check(args, 1, replacement=[]),
```

Comment:
Based on the `any.dims` schema [here](https://github.com/pytorch/pytorch/blob/534c73d478fc967cb3a6a29f5eada94bc4ce2c29/aten/src/ATen/native/native_functions.yaml#L727), the replacement should be `None`